client/components/sidebar.vue

```diff
@@ -119,22 +223,27 @@
           if (!el) {
             return null;
           }
-          return Object.assign({}, c, {highlightTop: getOffset(el).top});
+          return Object.assign({}, c, {
+            highlightTop: getOffset(el).top,
+            showDetails: false,
+            showAddCommentForm: false,
+            showAllReplies: c.replies.length <= 1,
+            hasManyReplies: c.replies.length > 1,
+            isReply: c.replyTo === null,
+            focus: false,
+          });
         }).filter((c) => {
           return c;
         }).sort((a, b) => {
           // We sort these values in order to place the comments correctly.
           // For example, you could have a newer comment positioned on top
           // of an older one, so we have to sort the positions accordinglys
           // before Vue tries to render them.
-          if (a.highlightTop > b.highlightTop) {
-            return 1;
-          }
-          else if (a.highlightTop < b.highlightTop) {
-            return -1;
+          if (a.highlightTop !== b.highlightTop) {
+            return a.highlightTop - b.highlightTop;
           }
           else {
-            return 0;
+            return a.createdAt - b.createdAt;
```

Comment:
You do not have to change this, but just to let you know, this can be further simplified to `a.highlightTop - b.highlightTop || a.createdAt - b.createdAt`.